### PR TITLE
[FTAMS102020] CeydaUlubas 

### PR DESCRIPTION
**Pull request by ceydaulubas**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -13,13 +13,32 @@ app.set('views', path.join(__dirname, 'views'));
...
 
-// ...
 
 // Add the route handlers here:
 
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers()
+    .then(beersFromApi => {
+      res.render('beers', {beersFromApi});
+      //console.log('Beers from the database: ', beersFromApi)
```
Comment (wax-bear) [inline app.js:25]: The route and the API call and its handler look good.
Do you still need this commented log?

File: app.js [hunk 2]
```diff
@@ -13,13 +13,32 @@ app.set('views', path.join(__dirname, 'views'));
...
+    })
+    .catch(error => console.log(error));   
+});
+
 app.get('/', (req, res) => {
```
Comment (wax-bear) [inline app.js:30]: As this is the index, what about placing this route above the "/beers" route?

File: app.js [hunk 3]
```diff
@@ -13,13 +13,32 @@ app.set('views', path.join(__dirname, 'views'));
...
   res.render('index');
 });
 
+app.get('/random-beers', (req, res) => {
+  punkAPI.getRandom()
+  .then(randombeerFromAPI => {
+    //console.log('Beers from the database: ', beersFromApi)
```
Comment (wax-bear) [inline app.js:37]: Same question here regarding the log.

File: views/beers.hbs [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<ul>
+    {{#each beersFromApi}}
+     <li>{{this.name}}</li>
```
Comment (wax-bear) [inline views/beers.hbs:3]: Could more be rendered here?
The iteration mentions rendering "Display an image, name, description and tagline".

File: views/random-beers.hbs [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<div class="randomBeer">
+    {{#each randombeerFromAPI }}
+    {{> beerpartial this}}
```
Comment (wax-bear) [inline views/random-beers.hbs:3]: Nice!

File: views/random-beers.hbs [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+    <p>{{this.food_pairing}}</p>
```
Comment (wax-bear) [inline views/random-beers.hbs:4]: Could these also be included in the partial?

File: views/random-beers.hbs [hunk 7]
```diff
@@ -0,0 +1,7 @@
+<div class="randomBeer">
```
Comment (wax-bear) [inline views/random-beers.hbs:1]: Could this page be styled according to the example in the iteration?

**Review by wax-bear — COMMENTED** [review]
Good stuff Ceyda!
Please check out my comments for potential improvements.

File: app.js (hunk 1 above)
Comment (ceydaulubas) [reply app.js:25]: Yeah, it's not required to use, I can delete commented logs.

File: app.js (hunk 2 above)
Comment (ceydaulubas) [reply app.js:30]: of course, I will share the corrected form with you again.

File: app.js (hunk 3 above)
Comment (ceydaulubas) [reply app.js:37]: deleted commented logs

File: views/beers.hbs (hunk 4 above)
Comment (ceydaulubas) [reply views/beers.hbs:3]: I added "{{>beerpartial this}}". Could you please check it?

File: views/random-beers.hbs (hunk 7 above)
Comment (ceydaulubas) [reply views/random-beers.hbs:1]: I'll fix the style.

**Pull request closed without merging**